charts/pega/README.md

```diff
@@ -642,14 +646,19 @@ installer:
   adminPassword: "ADMIN_PASSWORD"
 ```
 
-### Upgrade
+### Upgrades and patches
+
+To **upgrade Pega Platform software** deployed in a Kubernetes environment, you must download the latest Pega software from  [Pega Digital Software Delivery](https://community.pega.com/digital-delivery) and then use the appropriate upgrade guide:
+
+- Use the appropriate Pega Platform deployment Upgrade Guide available from the page, [Stay current with Pega](https://community.pega.com/upgrade).
+- Use the latest Pega application software Upgrade Guide, which is separate from Pega Platform software. You can locate the appropriate upgrade guide for your installed application from the page, [All Products](https://community.pega.com/knowledgebase/products).
 
-For upgrades of the Pega platform, you must specify the installer Docker image and the type of upgrade to execute.
+To **apply a Pega Platform patch** with zero downtime to your existing Pega platform software, you must use `installer.upgrade.upgradeType "out-of-place"` option. For step-by-step guidance to apply a Pega Platform patch, see the Pega-provided runbook, [Patching Pega Platform in your deployment](/docs/patching-pega-deployment.md)
 
 Upgrade type    | Description
 ---             | ---
-`in-place`      | An in-place upgrade will upgrade both rules and data in a single run.  This will upgrade your environment as quickly as possible but will result in downtime.
-`out-of-place`  | An out-of-place upgrade involves more steps to minimize downtime.  It will place the rules into a read-only state, then migrate the rules to a new schema. Next it will upgrade the rules to the new version. Lastly it will separately upgrade the data.
+`in-place`      | An in-place upgrade will upgrade both rules and data in a single run.  An in-place upgrade process will upgrade your environment as quickly as possible but pauses the deployment while and the process upgrades all of the rules of your Pega application.
```

Comment:
For 'in-place' upgrade , can you please revert your statements to use the older description. That clear says it requires down time for upgrade

Reply:
Sure. I updated the statement slightly to match what we say about in place upgrades for on-premises:
An in-place upgrade will upgrade both rules and data using a single process. This process involves system downtime, which means you cannot use the system until you complete the upgrade.